test/functionalTest/cases/0981_POST_v2_attributes/append_only_error_if_exists.test

```diff
@@ -41,7 +41,7 @@ brokerStart CB
 echo "01. Create E1"
 echo "============="
 payload='{ "attr1": 1 }'
-orionCurl --url /v2/entities/E1 --payload "$payload" --json
+orionCurl --url /v2/entities/E1 --payload "$payload" --json --noPayloadCheck
```

Comment:
So, I removed those changes (most probably inserted by myself) and it works just fine.
Fixed in b70841c